onnxruntime/core/graph/contrib_ops/contrib_defs.cc

```diff
@@ -2964,7 +2964,96 @@ It's an extension of Gelu. It takes the sum of input A and bias input B as the i
           ctx.getOutputType(0)
               ->CopyFrom(input_type->optional_type().elem_type());
           });
-  
+
+  static const char* GridSampler_ver1_doc = R"DOC(
```

Comment:
should we name this GridSample as opposed to GridSampler

Reply:
In PyTorch, the implementation name is GridSampler. Is better that we name GridSample instand of Gridsampler？ 
https://github.com/pytorch/pytorch/issues/41957
https://github.com/pytorch/pytorch/blob/master/aten/src/ATen/native/GridSampler.h